packages/algob/test/lib/tx.ts

```diff
@@ -143,11 +141,11 @@ describe("ASA modify fields", () => {
    * Verifies correct asset fields are sent to network
    * @param rawTxns rawTxns Signed transactions in Uint8Array
    */
-  function checkTx (rawTxns: Uint8Array | Uint8Array[]): Promise<ConfirmedTxInfo> {
+  function checkTx (rawTxns: Uint8Array | Uint8Array[]): Promise<modelsv2.PendingTransactionResponse> {
     if (isArray(rawTxns)) {
       // verify here if group tx
     } else {
-      const tx: Transaction = decodeSignedTransaction(rawTxns).txn;
+      const tx: any = decodeSignedTransaction(rawTxns).txn;
```

Comment:
was `Transaction` incorrect here?

Reply:
previoulsy it was, i saw there is updated commit in 1.10.0 , i will push the fixes after this and update scale-it.

Reply:
will we update this from `any`?